students/doyeon/users/views.py

```diff
@@ -1,3 +1,52 @@
-from django.shortcuts import render
+import json
+import re
+
+from django.http import JsonResponse
+from django.views import View
+from django.utils import timezone
+
+from .models import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+
+            name            = data["name"]
+            email           = data["email"]
+            password        = data["password"]
+            phone_number    = data["phone_number"]
+
+            regx_email      = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            regx_password   = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
+
+            if check_regx(regx_email, email):
+                if User.objects.filter(email=email).exists():
+                    raise ValueError("EXISTED E_MAIL")
+            if check_regx(regx_password, password):
+                if User.objects.filter(phone_number=phone_number).exists():
+                    raise ValueError("EXISTED PHONE_NUMBER")
+
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+        except ValueError as e:
+            return JsonResponse({"message": f"{e}"}, status=400)
+
+        else:
+            user = User.objects.create(
```

Comment:
```suggestion
        User.objects.create(
```
else: 는 애초에 불필요해보입니다.
추가로 create 한 결과를 user에 저장할 필요가 없어보입니다. user 객체가 재활용되지 않기 때문입니다!